### PR TITLE
feat(opentrons-ai-client): send chat history instead user prompt

### PR DESCRIPTION
**Pull request by koji**
# Overview
send chat history to the backend instead only user prompt.

The currently opentrons-ai-client sends userPrompt (string) to the backend but we need to send the entire chat history to align with the application on HuggingFace.
https://huggingface.co/spaces/Opentrons/OpentronsAI-v0.0.5/blob/main/flexchatinterface.py
The sample data is in the ticket.

close AUTH-407 AUTH-414

# Test Plan

# Changelog
- add a new atom chatHistoryAtom to store chatHistory

# Review requests

# Risk assessment
low

### CODE REVIEW

**Comment by koji** [comment]
- [ ] need to deploy the latest backend
- [ ] need to check backend log

File: opentrons-ai-client/src/molecules/InputPrompt/index.tsx [hunk 1]
```diff
@@ -63,8 +68,12 @@ export function InputPrompt(): JSX.Element {
             audience: 'sandbox-ai-api',
           },
         })
+        const message: Chat = {
+          role: 'user',
+          content: prompt,
+        }
         const postData = {
-          message: prompt,
+          messages: [...chatHistory, message],
```
Comment (Elyorcv) [inline opentrons-ai-client/src/molecules/InputPrompt/index.tsx:76]: why three dots in front of `chatHistory`?

**Comment by Elyorcv** [comment]
@koji I think you also need to modify server function such that it accepts list rather currently string?

File: opentrons-ai-client/src/molecules/InputPrompt/index.tsx (hunk 1 above)
Comment (koji) [reply opentrons-ai-client/src/molecules/InputPrompt/index.tsx:76]: destructuring

**Comment by koji** [comment]
> Reference

wait, server is not updated yet?
the format you shared in the Jira ticket might be different?

**Comment by Elyorcv** [comment]
> > Reference
> 
> wait, server is not updated yet? the format you shared in the Jira ticket might be different?

If you see lines 10-11 at `function.py` you see `ChatRequest`. I assume that we need to change this one as well to account for the changes you are making. @y3rsh may comment on this.

**Comment by koji** [comment]
This will be resolved by https://github.com/Opentrons/opentrons/pull/15288

**Pull request closed without merging**